0.5.0 (#742)
atoptima/Coluna.jl: README.md

active development.

- ![Stable](https://img.shields.io/badge/-stable-brightgreen) No stable feature at the moment
- ![Beta](https://img.shields.io/badge/-beta-green) Features that work but still in development:
  - Branch-and-price-and-cut algorithm
  - Cuts generation
  - Column generation algorithm
  - Dantzig-Wolfe decomposition
  - Pricing callback
  - Robust cut callback
  - Stabilization
  - Strong-branching

> - ![Alpha](https://img.shields.io/badge/-alpha-yellow) Features that should work. Structural work is done but it has bugs and may be not performant:

Author: ```suggestion
- ![Alpha](https://img.shields.io/badge/-alpha-yellow) Features that should work. Structural work is done but these features may have bugs and be not performant:
```
